Gopkg.toml

```diff
@@ -55,11 +55,3 @@
 [prune]
   go-tests = true
   unused-packages = true
-
-[[constraint]]
```

Comment:
Why remove these constraints? We hit issues in the past when these versions were not exactly lined up. Unless there's a good reason to remove these, I'd rather be safe than sorry and leave them here.

Reply:
They were indirect dependencies, my 6th sense felt you had issues in the past, but i did a try :) I will remove this from my PR thanks.